ceph.spec.in

```diff
@@ -121,6 +121,7 @@ BuildRequires:	pkgconfig
 BuildRequires:	python
 BuildRequires:	python-devel
 BuildRequires:	python-nose
+BuildRequires:	python-prettytable
```

Comment:
Well, I assume the same thing applies as for Debian, in that the package needs to be installed for make check to work.  Since the make check builds don't install the built packages to pull in prettytable via Requires, there's nothing else to do it that I know of.  (if there is some sort of "make check dependencies" mechanism I'm happy to learn about it.)